app/src/main/java/ch/epfl/sweng/swenggolf/offer/ListOwnOfferActivity.java

```diff
@@ -13,6 +13,9 @@
 import ch.epfl.sweng.swenggolf.database.Database;
 import ch.epfl.sweng.swenggolf.database.ValueListener;
 
+/**
+ * Fragment which displays only the user related offers.
```

Comment:
I think using related here is ambiguous, as we could think that offer where the user commented are related offers. But it only shows user own offers.